### PR TITLE
feat: add property "enableCopy" to TextField/TextArea

### PR DESCRIPTION
**Pull request by jquick-axway**
**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-25705

**Summary:**
- Added new boolean property "enableCopy" to `TextField`/`TextArea`.
- Defaults to `true`.
- Setting to `false` will do the following...
  * Remove menu items: `copy`, `cut`, `share`
  * Ignores `Command+C` and `Command+X` keyboard operations.
- Intended to improve security by preventing personal info from being copied.

**Note:**
On Android, we can't remove the `copy` and `cut` options from the virtual keyboard's "Text Editing" section or the landscape fullscreen context menu... but we can block the copy/cut actions from working.

---
**TextField Test:**
_(Run below test on iOS, Android 5.0, and Android 11.)_
1. Build and run `TextFieldDisableCopyTest.js` attached to [TIMOB-25705](https://jira.appcelerator.org/browse/TIMOB-25705).
2. Tap on the 1st text field, select the text, and tap-and-hold.
3. Verify context menu does show "copy" and "cut" options.
4. Tap on the 3rd text field, select the text, and tap-and-hold.
5. Verify context menu does **NOT** show "copy", "cut", or "share" options.
6. Tap on the 4th text field (it's a password field), select the text, and tap-and-hold.
7. Verify context menu does **NOT** show "copy", "cut", or "share" options.

**TextArea Test:**
_(Run below test on iOS, Android 5.0, and Android 11.)_
1. Build and run `TextAreaDisableCopyTest.js` attached to [TIMOB-25705](https://jira.appcelerator.org/browse/TIMOB-25705).
2. Tap on the 1st text area, select the text, and tap-and-hold.
3. Verify context menu does show "copy" and "cut" options.
4. Tap on the 2nd text area, select the text, and tap-and-hold.
5. Verify context menu does **NOT** show "copy", "cut", or "share" options.

**Bonus Test:**
There is a `TextFieldKeyboardAdvancedTest.js` script attached to [TIMOB-25705](https://jira.appcelerator.org/browse/TIMOB-25705) which makes it easy to test various combinations of text field options for ad-hoc testing.

### CODE REVIEW

File: iphone/Classes/TiUITextArea.m [hunk 1]
```diff
@@ -218,6 +233,11 @@ - (void)setEnabled_:(id)value
   [(UITextView *)[self textWidgetView] setEditable:_trulyEnabled];
 }
 
+- (void)setEnableCopy_:(id)value
```
Comment (vijaysingh-axway) [inline iphone/Classes/TiUITextArea.m:236]: You can move this API inside TiUITextWidgetProxy.

File: iphone/Classes/TiUITextField.m [hunk 2]
```diff
@@ -341,6 +354,11 @@ - (void)setEnabled_:(id)value
   [(TiTextField *)[self textWidgetView] setEnabled:_trulyEnabled];
 }
 
+- (void)setEnableCopy_:(id)value
```
Comment (vijaysingh-axway) [inline iphone/Classes/TiUITextField.m:357]: You can move this API inside TiUITextWidgetProxy.

File: iphone/Classes/TiUITextArea.m (hunk 1 above)
Comment (jquick-axway) [reply iphone/Classes/TiUITextArea.m:236]: But the implementation is different between `TiUITextArea` and `TiUITextField`. This function operates on `TiUITextViewImpl` and the other function operates on `TiTextField`.

**Review by garymathews — APPROVED** [review]
Android CR: **PASS**

**Review by vijaysingh-axway — APPROVED** [review]
iOS changes looks good!

**Comment by lokeshchdhry** [comment]
@jquick-axway , The changes doe not seem to be working on Android 5.1 API 22. 
For copy enabled textfield or text area I do not get a copy option, only paste. So basically I cant copy the text.

**Comment by jquick-axway** [comment]
@lokeshchdhry, are you sure you don't see a "copy" option in the top toolbar? The popup menu won't show the "copy" option for me either on Android 5.x, but the same is true in Google's own apps like Messaging. Basically, we can't control where the Android OS puts these options (in a popup or toolbar), but they are there for me.

**Comment by lokeshchdhry** [comment]
@jquick-axway , Alright makes sense. I see the copy option in the tool bar. Was not sure if popup should be an option.

**Comment by lokeshchdhry** [comment]
FR Passed.

**Pull request merged**